rules.go

```diff
@@ -39,6 +41,16 @@ func IsPermissionResource(str string) bool {
 	return rxResourcePermission.MatchString(str)
 }
 
+func IsAlphaNumUnicode(str string) bool {
+	for _, char := range str {
+		if isValid := unicode.In(char, unicode.M, unicode.Nl, unicode.L, unicode.N); !isValid {
```

Comment:
just want to make sure here. will this function check if the unicode contain letter, number, and mark? please CMIIW

Reply:
Yes, it will check if the input string is a member of one of the rule

Reply:
sure thing. Thanks :+1: